.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Oh no, a bug. It happens. Thanks for reporting an issue with NewPipe
+
+Use this template to notify us if you found a bug.
+
+To make it easier for us to help you please enter detailed information below.
+
+Please note, we only support the latest version of NewPipe and master branch. Please make sure to upgrade & recreate the issue on the latest version prior to opening an issue. The release page (https://github.com/TeamNewPipe/NewPipe/releases/latest) is a good start, make sure its version is the same as in your app (to check your version, open the left drawer and click on "About").
+
+P.S.: Our [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) might be a nice document to read before you fill out the report :)
```

Comment:
```suggestion
P.S.: Our [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) might be a nice document to read before you fill out the report. :)
```

Reply:
Imo, an emoji replaces the punctuation

Reply:
What? When did that become the norm? Why wasn't I called to that meeting? This is breach of protocol!

Anyway, it looks okay in _some_ cases, but not in others.

Reply:
so, is this one of the cases?

Reply:
It looks fine _here_, but, you know, consistency.

Reply:
"we" arent using more emojis in the documents then the two cases you mentioned

Reply:
So. Are we just sticking with it?

Reply:
Your wish. I don't have enough faith in the lord left to convince you either way. YOU HAVE SHAKEN ME.

Reply:
uhhmmm. sure. Im just, uh. gonna go over there. jesus.

Reply:
**unblinking stare**